### DISCUSSION TITLE
Image Component support for "avif" image format

### DISCUSSION THREAD

**Discussion by Svatex** [diegomura/react-pdf#2776] [Q&A]
Hi guys,
We would like to use the `avif` image format for our project. I didn't find any official list of supported image formats for `<Image/>` component, there is just mention in docs that buffer png/jpg will be used if not specified, so it sounds like some other formats are supported too. On the other hand, in types, it seems that only png/jpg is supported. Is there any way to make it work?
Thanks!

<img width="573" alt="image" src="https://github.com/diegomura/react-pdf/assets/65604288/7fc0a13c-7a22-431d-ac7c-0bdec358a18f">

**Comment by rehanshaikymts** [comment]
Facing same issue, all the images we have stored are in avif format, IG I need to write a route in backend to get the images from cdn, and convert it to other formats. 😪
